tests/test_end_to_end_employee_skype_message.py

```diff
@@ -0,0 +1,98 @@
+"""
+This End to end test employee skype message covers following:
+Setup- Purging SQS queue
+Step 1: Trigger employee message lambda
+Step 2: Print message from cloudwatch logs
+Step 3: Verify message with skype-listner sqs queue and culture file
+
+"""
+import os
+import sys
+import time
+import asyncio
+import logging
+import unittest
+import pytest
+from pythonjsonlogger import jsonlogger
+import tests.helpers.asyncio_helper
+import tests.helpers.cloudwatch_helper
+import tests.helpers.filter_message_helper
+import tests.helpers.lambda_helper
+import tests.conf.cloudwatch_configuration_conf as cloudwatch_conf
+import tests.conf.sqs_utilities_conf as queue_url_conf
+import tests.conf.lambda_configuration_conf as lambda_conf
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
Is this step required?
Adding `conftest.py` to a directory adds that location to sys.path